diff --git a/src/librustdoc/core.rs b/src/librustdoc/core.rs
index 375f833831..942dba29e6 100644
--- a/src/librustdoc/core.rs
+++ b/src/librustdoc/core.rs
@@ -202,7 +202,7 @@ pub(crate) fn create_config(
         diagnostic_width,
         libs,
         externs,
-        mut cfgs,
+        cfgs,
         check_cfgs,
         codegen_options,
         unstable_opts,
@@ -220,7 +220,7 @@ pub(crate) fn create_config(
     render_options: &RenderOptions,
 ) -> rustc_interface::Config {
     // Add the doc cfg into the doc build.
-    cfgs.push("doc".to_string());
+    // cfgs.push("doc".to_string());
 
     // By default, rustdoc ignores all lints.
     // Specifically unblock lints relevant to documentation or the lint machinery itself.
